Payments/UserInterface/08a_Connections.md

```diff
@@ -0,0 +1,201 @@
+[!!Establish connections to a payment service provider](../UserInterface/02a_ListPaymentsAndCaptures.md)
+
+# CONNECTIONS (Settings)
```

Comment:
## Connections

Header sind in der Regel kleingeschrieben. Hier würde ich (Settings) weglassen. Bei Fulfillment habe ich es dazu geschrieben, weil es ein Connections Reiter auch bei Logging gibt, also zu unterscheiden. Daher auch: List (Authorizations), List (Refunds) usw.